lib/umbra.rb

```diff
@@ -1,20 +1,14 @@
 # frozen_string_literal: true
 
-require "typhoeus"
+require "zeitwerk"
+require "rack"
 require "redis"
-require "multi_json"
 require "concurrent"
 
+Zeitwerk::Loader.for_gem.setup
+
 module Umbra
-  autoload :Config, "umbra/config"
-  autoload :Encoder, "umbra/encoder"
-  autoload :Middleware, "umbra/middleware"
-  autoload :Publisher, "umbra/publisher"
-  autoload :Subscriber, "umbra/subscriber"
-  autoload :RequestBuilder, "umbra/request_builder"
-  autoload :ShadowRequester, "umbra/shadow_requester"
-  autoload :SynchronousPublisher, "umbra/synchronous_publisher"
-  autoload :Version, "umbra/version"
+  autoload :Pb, "umbra/pb/umbra_pb"
```

Comment:
Need to autoload `Pb` because the protobuf compiler generated filename doesn't conform to `zeitwerk`